docs(tutorials): content review week 35/2023 (#2075)
scaleway/docs-content: tutorials/zammad-ticketing/index.mdx

## Zammad - Overview

> [Zammad](https://zammad.org/) stands as an open-source helpdesk system equipped with capabilities to oversee customer interactions across various communication channels, encompassing telephone, Facebook, Twitter, chat, and emails. Notably, the software is entirely open-source and is distributed under the GNU AFFERO General Public License version 3 (GNU AGPLv3).

Reviewer: ```suggestion
[Zammad](https://zammad.org/) is an open-source helpdesk system that allows you to oversee customer interactions across various communication channels, encompassing telephone, Facebook, Twitter, chat, and emails. Notably, the software is entirely open-source and is distributed under the GNU AFFERO General Public License version 3 (GNU AGPLv3).
```
